Sources/Kommunicate/Classes/Extensions/KMConversationService+ConversationFeedback.swift

```diff
@@ -108,4 +108,55 @@ extension KMConversationService {
             }
         }
     }
+    
+    func submitFiveStarFeedback(
```

Comment:
Can't we use existing function for api call

Reply:
the parameter type for Feedback is different in both. As one is taking in enum and this one is taking in integer.